gh-actions/e2e/action.yaml

```
@@ -70,15 +70,18 @@ runs:
         echo "::endgroup::"
 
     - name: Restore images from the cache
-      uses: submariner-io/shipyard/gh-actions/restore-images@devel
+      uses: submariner-io/shipyard/gh-actions/restore-images@release-0.17
       with:
         cache: ${{ inputs.cache }}
         working-directory: ${{ inputs.working-directory }}
 
     - name: Run E2E deployment and tests
       shell: bash
       run: |
-        k8s_version=${{ inputs.k8s_version }} &&
+        k8s_version=${{ inputs.k8s_version }};
+        if [ "$k8s_version" = k8s-latest ]; then k8s_version=1.33; fi; \
```

```suggestion
        if [ "$k8s_version" = k8s-latest ]; then k8s_version=1.29; fi; \
```

Or did you intend for this to be 1.33?

---

I didn’t see this in the backport process… [Our policy](https://submariner.io/development/building-testing/ci-maintenance/#kubernetes-versions) doesn’t specify release branches. However since kind PRs aren’t backported we probably don’t have support for the latest versions on older branches!

---

I think I follow what you said. I don't think we need to backport these PRs since since we don't/haven't changed K8s versions in release branches (unless we want to start doing so).